.travis.yml

```diff
@@ -1,5 +1,23 @@
 language: node_js
 node_js: node
+addons:
+  apt:
+    packages:
+      # Japanese
+      - language-pack-ja
+      - japan*
+      # Chinese
+      - language-pack-zh*
+      - chinese*
+      # Korean
+      - language-pack-ko
+      - korean*
+      # Fonts
+      - fonts-arphic-ukai
+      - fonts-arphic-uming
+      - fonts-ipafont-mincho
+      - fonts-ipafont-gothic
+      - fonts-unfonts-core
```

Comment:
What are these?

Reply:
the OS travis uses does not have asian language fonts, so the visual diffs just had empty square for these languages. This was what google said needed to be added.